docs/pages/appbars/index.vue

```diff
@@ -0,0 +1,188 @@
+
+      <p>
+        We have four types of app bars: <em>primary bars</em>, <em>back arrow bars</em>, <em>form bars</em>, and <em>bottom bars</em>
+      </p>
+
+      <h3>Primary bars</h3>
+      <p>Primary bars are placed on top level pages in Kolibri. They may house subtabs that lead to different sections of the application</p>
+      <DocsShow>
+        <img class="modal-img" src="./primary-1.png">
+      </DocsShow>
+      <DocsShow>
+        <img class="modal-img" src="./primary-2.png">
+      </DocsShow>
+
+      <h3>Back arrow bars</h3>
+      <p>Back arrow topbars allow users to navigate back to the previous page</p>
+      <DocsShow>
+        <img class="modal-img" src="./back-1.png">
+      </DocsShow>
+
+      <h3>Form bars</h3>
+      <p>Form bars allows users to dismiss a full-screen form and land on the previous page</p>
+      <DocsShow>
+        <img class="modal-img" src="./form-1.png">
+      </DocsShow>
+
+      <h3>Bottom bars</h3>
+      <p>Bottom bars are placed on pages that house editing, content selection, or multi-step processes. They may contain buttons that allow the user to submit specific action(s) or continue with the process. Bottom bars can also contain contextual information and error messaging relevant to the task</p>
+      <DocsShow>
+        <img class="modal-img" src="./bottom-1.png">
+      </DocsShow>
+    </DocsPageSection>
+
+    <DocsPageSection title="Layout" anchor="#layout">
+      <ul>
+        <li>Place navigation icons on the far left</li>
+        <li>Place titles to the right of the navigation icon</li>
+        <li>Place contextual icons to the far right</li>
+      </ul>
+    </DocsPageSection>
+
+    <DocsPageSection title="Titles" anchor="#titles">
+      <p>Appbars must have titles that describe the current page</p>
+      <ul>
+        <li>Titles can describe a feature a user is engaged in (e.g. Create Quiz)</li>
+        <li>Titles can describe a high-level section of an app (e.g. Learn, Coach)</li>
+        <li>Titles can describe an application being used (e.g. Kolibri Studio)</li>
+        <li>Titles should never include names of user generated text (e.g. resource title or username)</li>
+      </ul>
+    </DocsPageSection>
+
+    <DocsDoNot>
+      <template v-slot:do>
+        <img src="./do-1.png">
+        <p class="do-dont">
+          Use general titles in app bars
+        </p>
+      </template>
+      <template v-slot:not>
+        <img src="./dont-1.png">
+        <p class="do-dont">
+          Use user-generated text in app bars
+        </p>
+      </template>
+    </DocsDoNot>
+
+    <DocsPageSection title="Exceptions" anchor="#exceptions">
+      <p>Purposeful exceptions can be made to incorporate special elements into top bars</p>
+    </DocsPageSection>
+
+    <DocsDoNot style="display: inline-block;">
+      <template v-slot:do>
+        <img src="./exception-1.png">
+        <p class="do-dont">
+          Appbar color may change to white for critical workspaces
```

Comment:
What does this actually mean? Can we define "critical"? I don't think the guidance is actionable